app/src/main/java/io/intrepid/contest/screens/adminstatus/AdminStatusPresenter.java

```diff
@@ -0,0 +1,75 @@
+package io.intrepid.contest.screens.adminstatus;
+
+import android.support.annotation.NonNull;
+
+import java.util.List;
+
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BasePresenter;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Entry;
+import io.reactivex.disposables.Disposable;
+import timber.log.Timber;
+
+class AdminStatusPresenter extends BasePresenter<AdminStatusContract.View> implements AdminStatusContract.Presenter {
+
+    private List<Entry> entries;
+    private int numEntriesMissing = 0;
+
+    AdminStatusPresenter(@NonNull AdminStatusContract.View view,
+                         @NonNull PresenterConfiguration configuration) {
+        super(view, configuration);
+        fetchSubmittedEntries();
+    }
+
+    @Override
+    public void onViewCreated() {
+        super.onViewCreated();
+        view.showAwaitSubmissionsIndicator(true);
+        view.showJudgingStatusIndicator(true);
+        view.showEndOfContestIndicator(true);
+    }
+
+    void fetchSubmittedEntries() {
+        String contestId = String.valueOf(persistentSettings.getCurrentContestId());
+        Disposable disposable = restApi.getContestDetails(contestId)
+                .compose(subscribeOnIoObserveOnUi())
+                .subscribe(response -> {
+                    Timber.d("Received response " + response.contest.getTitle());
+                    this.entries = response.contest.getEntries();
+                    view.showSubmittedEntries(entries);
+                }, throwable -> {
+                    Timber.d("API error retrieving contest details: " + throwable.getMessage());
+                    view.showMessage(R.string.error_api);
+                });
+        disposables.add(disposable);
+    }
+
+//    void fetchContestStatus() {
```

Comment:
let's pull this out if we're not using it. We don't want to leave dead code in the project.